templates/admin/base_site.html

```diff
@@ -10,7 +10,7 @@ <h1 id="site-name">
             <a href="{% url 'admin:index' %}">{{ site_header|default:_('Django administration') }}</a>
         </h1>
         <div style="position: fixed;right: 20px;bottom: 20px;color: #00000014; display: grid;font-weight: bold; pointer-events: none;">
-            {% if branch != "develop" or branch != "master" %}
+            {% if branch != "develop" and branch != "master" %}
```

Comment:
Лучше так! отрицание всегда тяжелее читать чем просто сравнение
```suggestion
            {% if branch == "develop" or branch == "master" %}
```

Reply:
Не вдумывался и могу ошибаться, но комментарий на глаза попался.

А тут точно делаете что нужно? Сравнение, конечно, лучше понимается чем отрицание, но ведь выражение поменялось на строго противоположное.

Если правильно понимаю теперь плашку будем показывать только на ветках мастер и девелоп. Так точно задумано?

Reply:
дааа, я комитнул приложение и сразу понял что работать не будет) я уже просто попозже сегодня верно сделаю